tools/shared.py

```diff
@@ -2658,6 +2659,16 @@ def path_to_system_js_libraries_for_settings(link_settings):
       system_js_libraries += ['library_egl.js', 'library_glut.js', 'library_gl.js']
     return [path_from_root('src', x) for x in system_js_libraries]
 
+  @staticmethod
+  def get_binaryen_feature_flags():
+    # start with the MVP features, add the rest as needed
+    ret = ['--mvp-features']
+    if Settings.USE_PTHREADS:
+      ret += ['--enable-threads']
+    if Settings.SIMD:
+      ret += ['--enable-simd']
+    return ret
```

Comment:
What is our story for enabling the other features, nontrapping float-to-int conversions and mutable globals?